exporter/exporterhelper/queued_retry.go

```diff
@@ -127,11 +142,24 @@ func newQueuedRetrySender(fullName string, qCfg QueueSettings, rCfg RetrySetting
 }
 
 // start is invoked during service startup.
-func (qrs *queuedRetrySender) start() {
+func (qrs *queuedRetrySender) start() error {
 	qrs.queue.StartConsumers(qrs.cfg.NumConsumers, func(item interface{}) {
 		req := item.(request)
 		_ = qrs.consumerSender.send(req)
 	})
+
+	// Start reporting queue length metric
+	if qrs.cfg.Enabled {
+		metricproducer.GlobalManager().AddProducer(r)
```

Comment:
This probably better to happen in a "init" func.

Reply:
I don't see a way to test it if we put it in init. Other tests are mixing it up during initialization of this package and I don't know a way to clean it up after

Reply:
nvm. moved it to init and set proper cleanup